src/osec.c

```diff
@@ -250,6 +267,114 @@ static bool create_cdb(int fd)
 	return retval;
 }
 
+static bool create_virtual_cdb(int fd)
```

Comment:
> If you have noticed, `osec` does not allow you to specify individual files in the database

That's why I've implemented this. In "virtual directory" mode the _entire_ database for a virtual directory (an arbitrary list of files) is contained in just a single file that can (and have to) be specified.